StudentProject/StudentData.java

```diff
@@ -0,0 +1,31 @@
+package StudentProject;
+
+class StudentData {
+    private int StudentID;
```

Comment:
The Java convention for fields is lowerCamelCase (https://www.techtarget.com/whatis/definition/lowerCamelCase).
Use _studentId_ instead of _StudentID_ ...